config/prow/plugins.yaml

```diff
@@ -357,6 +357,10 @@ milestone_applier:
     master: v0.4
     release-0.4: v0.4
     release-0.3: v0.3
+  kubernetes-sigs/controller-runtime:
+    master: v0.9.x
+    release-0.8: v0.8.x
+    release-0.7: v0.7.x
```

Comment:
Let's add 0.6 and 0.5? Or nah?

Reply:
Do we still consider them supported?

Reply:
Probably only for important security fixes

Reply:
updated it